### ISSUE TITLE
(suggestion) add moving enemies 

### ISSUE THREAD

**Issue by yasaman72** [jadijadi/riverraidrust#72]
It would be a nice challenge to shoot (and of course implement) moving enemies. There could be various types of enemies with different movement patterns (vertical, horizontal, or anything in between)

**Comment by jozefkonrad** [comment]
yeah - if i remember correctly, there were helicopters in the original game.

moving left and right, shooting at low range.
